packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -539,8 +539,8 @@ export class MarketOperationUtils {
         };
 
         // NOTE: For sell quotes input is the taker asset and for buy quotes input is the maker asset
-        const ethToTakerAssetRate = side === MarketOperation.Sell ? ethToInputRate : ethToOutputRate;
-        const ethToMakerAssetRate = side === MarketOperation.Sell ? ethToOutputRate : ethToInputRate;
+        const takerAssetPriceForOneEth = side === MarketOperation.Sell ? ethToInputRate : ethToOutputRate;
```

Comment:
Did we want to update the naming of `ethToInputRate` and `ethToOutputRate` as well? I'm a bit fuzzy on what we decided upon 🤔

Reply:
@kimpers 😂 yeah, I mean I like that new naming, it does make things clearer to me. So ok for `inputPriceForOneEth` and `outputPriceForOneEth`?

Reply:
@dorothy-zbornak @xianny @dekz

Reply:
Honestly I am even more confused by these new names. "Price" is weird to me here because it is usually expressed in a different, base token. Why not just `inputTokensPerEth`, etc.